src/platform/common/event.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
Please move this content into `src/platform/common/utils/events.ts`

Reply:
Updated, thanks!